docs/development.md

```diff
@@ -1,4 +1,4 @@
-# Development Environments
+e Development Environments
```

Comment:
Looks like this is an editing mistake :)

Reply:
oops, updated